index.html

```diff
@@ -0,0 +1,174 @@
+              sing up required
+            </p>
+            <ul>
+              <li><button class="lg">html</button></li>
+              <li><button class="lg">css</button></li>
+              <li><button class="lg">javaScript</button></li>
+            </ul>
+            <br />
+            <button class="s">See Project</button>
+          </section>
+        </li>
+        <li>
+          <section>
+            <h2>About Myself</h2>
+            <p>
+              I'm a software developer! I can help you built a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don't
+              hestiate to contact me.
+            </p>
+            <div class="Sosmed">
+              <h3>LET'S CONNECT</h3>
+              <div class="Sosmed-icon">
+                <ul>
+                  <li>
+                    <a class="social-icons" href=""
+                      ><img src="IMG/Twiters.svg" alt="Twiters"
+                    /></a>
+                  </li>
+                  <li>
+                    <a class="social-icons" href=""
+                      ><img src="IMG/LinkedIn.svg" alt="LinkedIn"
+                    /></a>
+                  </li>
+                  <li>
+                    <a class="social-icons" href=""
+                      ><img src="IMG/Medium.svg" alt="Medium"
+                    /></a>
+                  </li>
+                  <li>
+                    <a class="social-icons" href=""
+                      ><img src="IMG/Github.svg" alt="Github"
+                    /></a>
+                  </li>
+                  <li>
+                    <a class="social-icons" href=""
+                      ><img src="IMG/Angellist.svg" alt="Angellist"
+                    /></a>
+                  </li>
+                </ul>
+              </div>
+            </div>
+            <button class="s get-my-resume">Get my resume</button>
+          </section>
+        </li>
+        <li>
+          <section>
+            <h2>Languages</h2>
+            <div class="round-img-list">
+              <ul>
+                <li>
+                  <img src="IMG/js.svg" alt="" class="round-img" />
+                </li>
+                <li>
+                  <img src="IMG/html.svg" alt="" class="round-img" />
+                </li>
+                <li>
+                  <img src="IMG/css.svg" alt="" class="round-img" />
+                </li>
+              </ul>
+            </div>
+            <ul>
+              <li class="skills">
+                <p><b>Frameworks</b></p>
+                <img src="dropright" alt="" />
+              </li>
+              <li class="skills">
+                <p><b>Skills</b></p>
+                <img src="dropright" alt="" />
+              </li>
+            </ul>
+          </section>
+        </li>
+      </ul>
```

Comment:
- [ ] **You are using a `ul` tag to group the `My-Project` section and `About Myself` section. Please separate them into their own section tags as they are two different sections on the page.**